### PR TITLE
Master web clear cache aab

### PR DESCRIPTION
**Pull request by aab-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/views/basic/basic_model.js [hunk 1]
```diff
@@ -1022,8 +1034,13 @@ var BasicModel = AbstractModel.extend({
                 method: 'write',
                 args: [resIDs, { active: value }],
             })
-            .then(this.reload.bind(this, parentID));
-
+            .then(function () {
```
Comment (ged-odoo) [inline addons/web/static/src/js/views/basic/basic_model.js]: haha

**Comment by ged-odoo** [comment]
no commit to invalidate cache in sales channel?

**Comment by aab-odoo** [comment]
- edit sales channel, it changes the nocontent helper for the main crm lead view, it should apply even when no full refresh was made
AAB: i wasn't able to find the usecase
- add a column in sales crm kanban view => it changes  the form view (status bar: new option).  This is probably the case of many other kanban views. Not sure what to do for this one.
AAB: this is not stored in the view, so even without clearing the cache, the new column appears by replaying the action

**Comment by ged-odoo** [comment]
you're right, I cannot find the usecase anymore.

**Pull request merged**